web-app/web-frontend/script.js

```diff
@@ -1,193 +1,224 @@
 // NOTE: this example uses the chess.js library:
 // https://github.com/jhlywa/chess.js
 
-var board = null
-var game = new Chess()
-game.load("rnbqkbnr/ppp1pppp/8/4PP2/8/P7/1PpP2PP/RNBQKBNR b KQkq - 0 5");
-var $status = $('#status')
-var $pgn = $('#pgn')
-var $gan = $('#gan')
-
-function onDragStart (source, piece, position, orientation) {
+let board = null;
+const game = new Chess();
+const $status = $('#status');
+const $pgn = $('#pgn');
+const $gan = $('#gan');
```

Comment:
I assume these changes are from following the google style guide - nice! I am not familiar with js style so you can be our resident style expert :)